rust/otap-dataflow/crates/otap/src/otap_grpc/otlp/server.rs

```diff
@@ -83,6 +85,19 @@ impl SharedState {
     }
 }
 
+/// Common settings for OTLP receivers.
+#[derive(Clone, Debug)]
+pub struct Settings {
+    /// Maximum concurrency
+    pub max_concurrent_requests: usize,
```

Comment:
I believe this is the max concurrent requests per core instance of the receiver and not across all the receiver instances. It would be good to be clear about (preferably in the name of the setting).

Reply:
It is indeed a limit that applies per core.

Reply:
added "/// Maximum concurrent requests per receiver instance (per core).".